libs/ob-sql-parser/src/main/resources/obmysql/sql/OBLexer.g4

```diff
@@ -19,6 +19,324 @@ lexer grammar OBLexer;
 public boolean inRangeOperator = false;
 }
 
+// the below is what's new in OB 4.3.3.0
+UnderlineUTF8MB3
```

Comment:
should keep Lexer name style consistent?

Reply:
Yes, we better keep it consistent in theory, but there are many other tokens which also have the same problem and they are brought in previous versions. I prefer to keep the original this time.